.github/CODEOWNERS

```diff
@@ -57,6 +57,8 @@ xptifw/ @intel/llvm-reviewers-runtime
 
 # DPC++ tools
 llvm/ @intel/dpcpp-tools-reviewers
+clang/lib/CodeGen/BackendUtil.cpp @intel/dpcpp-cfe-reviewers @intel/dpcpp-tools-reviewers
+clang/include/clang/CodeGen/BackendUtil.h @intel/dpcpp-cfe-reviewers @intel/dpcpp-tools-reviewers
```

Comment:
Note that an approval from *one* of the groups would be enough to merge the patch. @elizabethandrews , can you please reply if you're still ok with it?

Reply:
Oh really? There's no way to require both?

Reply:
None that I'm aware of.

Reply:
I guess I am ok with it. Most of the time, changes in this file are to add passes and the FE change is minor. We just need to keep an eye out for larger changes I guess. @premanandrao are you ok with this?